lib/api/src/js/imports.rs

```diff
@@ -172,11 +171,17 @@ impl Imports {
         for (ns, exports) in namespaces.into_iter() {
             let import_namespace = js_sys::Object::new();
             for (name, ext) in exports {
-                js_sys::Reflect::set(&import_namespace, &name.into(), &ext.as_jsvalue(store))
-                    .expect("Error while setting into the js namespace object");
+                #[allow(unused_unsafe)]
+                unsafe {
```

Comment:
Why we are doing this? `#[allow(unused_unsafe)]` and then `unsafe {`. Doesn't make that much sense.
Perhaps I'm missing something?

Reply:
`js_sys::Reflect::get(...)` is `extern "C"`. My guess is that it's not unsafe depending were the rust code for the function itself is defined relative to the crate built.

Reply:
So, the `#[allow(unused_unsafe)]` / `unsafe { ... }` can be remove, it will build fine. But IDE like VSCode are not happy

![image](https://user-images.githubusercontent.com/3250792/210518290-28a1e08f-0e91-4a20-a162-95d011523355.png)

So I think it's better to keep it that way for now.

Reply:
Code now has comments that explain it.